RecoLocalCalo/HcalRecAlgos/interface/MahiFit.h

```diff
@@ -127,12 +127,14 @@ class MahiFit
 
   double getSiPMDarkCurrent(double darkCurrent, double fcByPE, double lambda) const;
   
-  mutable nnlsWorkspace nnlsWork_;
+  mutable MahiNnlsWorkspace nnlsWork_;
 
   //hard coded in initializer
   const unsigned int fullTSSize_;
   const unsigned int fullTSofInterest_;
 
+  int pedestalBX_;
```

Comment:
this was meant to be a constant, and from use cases it can even be a constexpr.
Please change, unless there are some arguments that this should be a variable value.